doc/vxlan/EVPN/EVPN_VxLAN_Multihoming.md

```diff
@@ -0,0 +1,1450 @@
+```
+sonic# show bgp l2vpn evpn next-hops
+VRF             IP              RMAC              #Paths     Base Path
+RED             10.10.10.3      44:38:39:22:01:bb 4          10.1.10.0/24
+RED             10.10.10.4      44:38:39:22:01:c1 4          10.1.10.0/24
+RED             10.10.10.2      44:38:39:22:01:af 4          10.1.10.0/24
+BLUE            10.10.10.3      44:38:39:22:01:bb 2          10.1.30.0/24
+BLUE            10.10.10.4      44:38:39:22:01:c1 2          10.1.30.0/24
+BLUE            10.10.10.2      44:38:39:22:01:af 2          10.1.30.0/24
+```
+
+
+### 3.5.5 Debug Commands
+
+Following FRR debug commands are available for EVPN VxLAN and Mulithoming debugging:
+
+```
+sonic(config)# log syslog debugging
+
+sonic(config)# debug bgp evpn mh
+  es     Ethernet Segment debugging
+  route  Route debugging
+sonic(config)# debug bgp zebra
+
+sonic(config)# debug zebra evpn mh
+  es     Ethernet Segment Debugging
+  mac    MAC Debugging
+  neigh  Neigh Debugging
+  nh     Nexthop Debugging
+sonic(config)# debug zebra vxlan
+sonic(config)# debug zebra kernel
+sonic(config)# debug zebra dplane
+sonic(config)# debug zebra fpm
+
+```
+
+### 3.5.6 REST API Support
+
+** Ethernet-segment configuration **
+```
+curl -X PATCH "https://SWITCH_IP:9090/restconf/data/openconfig-network-instance:network-instances/network-instance=default/evpn/ethernet-segments/ethernet-segment=PortChannel17" -H "accept: */*" -H "Content-Type: application/yang-data+json" -d "{\"openconfig-network-instance:ethernet-segment\":[{\"name\":\"PortChannel17\",\"config\":{\"esi-type\":\"TYPE_0_OPERATOR_CONFIGURED\",\"esi\":\"0017000000000000000a\"},\"df-election\":{\"config\":{\"preference\":201}}}]}"
+```
+
+** EVPN Multihoming global configuration **
+```
+curl -X PATCH "https://SWICH_IP:9090/restconf/data/openconfig-network-instance:network-instances/network-instance=default/evpn/evpn-mh/config" -H "accept: */*" -H "Content-Type: application/yang-data+json" -d "{\"openconfig-network-instance:config\":{\"startup-delay\":150,\"mac-holdtime\":200,\"neigh-holdtime\":250}}" 
+```
+
+
+
+# 4 Flow Diagrams
+
+-
+
+
+
+# 5 Error Handling
+No explicit error handling requirement introduced due to this feature.
+
+
+
+
+# 6 Serviceability and Debug
+Existing serviceability and debug applicable to this feature. No new debugging commands are added.
+
+
+
+# 7 Warm Boot Support
+Warm reboot for EVPN Multihoming feature will be supported on the platforms that support warm-boot. All of the newly added components and functionalities support warm-reboot processing.
+
+
+
+# 8 Scalability
+Below are the targeted EVPN Multihoming scale numbers on platforms providing EVPN Multihoming capability.
+
+| Metric                     | Scale | 
+|:---------------------------|:-------|
+| EVPN MH ES VTEP membership | 4      | 
+| ESIs in a site/PoD         | 1K     | 
+| ESIs on a VTEP             | 128    |
+| VNI per ESI                | 4K     |
+
+
+
+
+# 9 Unit Test
```

Comment:
EVPN MH will require a separate testplan documentation, please move this to a test plan submission

Reply:
Agreed. Removed from here for now.